### DISCUSSION TITLE
trying to play custom maps on webgl verson and a get a dl 404 error code need help

### DISCUSSION THREAD

**Discussion by FrostyXPP** [nzp-team/nzportable#913] [General]
when i go to drag the zip file for the map i see the download bar for 1 second and the it says can not download the it says dl error(404) 

To Reproduce
hit the tide key and then drag zip file to to the console 

I only tried this on webgl verson of the game

**Comment by Peter0x44** [comment]
Is there a reason you can't use the desktop version? I'm not sure we support loading maps that way.

**Comment by FrostyXPP** [comment]
I only have this crappy Chromebook that is outdated and does not get security anymore is there a chrome OS version of the game besides browser and is there any other way to port maps into the webgl version of the game?

**Comment by FrostyXPP** [comment]
here is a screenshot if that would help.
![Screenshot 2024-09-26 10 50 10](https://github.com/user-attachments/assets/0fce37dd-4adc-4674-b64c-759ff058356f)

**Comment by Peter0x44** [comment]
I'm not sure that is supported. Maybe someone else knows for sure.

**Comment by DiabloGane45** [comment]
Can you just drag the native map file into the game then type map (your map name) to load it up?

**Comment by DiabloGane45** [comment]
Hi, Swamp here just export the map files in a folder then drag and drop the bsp file into the web version, click install then yur dawn

**Comment by MotoLegacy** [comment]
converting this to a discussion since you once again didn't follow the issue template, and instead just completely deleted it
